sources/AppBundle/Controller/Website/TalksController.php

```diff
@@ -43,9 +44,18 @@ public function __construct(
         $this->algoliaFrontendApikey = $algoliaFrontendApikey;
     }
 
-    public function list(): Response
+    public function list(Request $request): Response
     {
+        $title = 'Historique des conférences de l\'AFUP';
+        if (isset($request->get('fR')['speakers.label'][0])) {
+            $title = 'Les vidéos de ' . $request->get('fR')['speakers.label'][0];
+        }
+        if (isset($request->get('fr')['event.title'][0])) {
```

Comment:
Histoire de pas rentrer dans les 2 if :

```suggestion
        else if (isset($request->get('fr')['event.title'][0])) {
```